data_ingestion/services/vector_db_client.py

```diff
@@ -36,27 +40,34 @@ def data_upsertion(slugs, func, batch_size=100) -> None:
             ),
         )
 
-        logger.info("Batch upserted")
+        logger.debug("Batch upserted")
+
+    logger.info("Data upsertion completed")
+
 
-    logger.info("Text upserted")
+def create_url_text_map(record: dict[str, str]) -> list[dict[str, str]]:
+    if record is None:
+        return []
+    chunks = chunk_text(text=record["text"])
+    return list(map(lambda text: {"url": record["link"], "text": text}, chunks))
 
 
-def recreate_collection():
+def recreate_collection() -> bool:
     return client.recreate_collection(
         collection_name=settings.qdrant_collection,
         vectors_config=models.VectorParams(distance=models.Distance.COSINE, size=1536),
     )
 
 
-def get_collection():
+def get_collection() -> None:
```

Comment:
A fairly minor point, and one I'm perfectly happy to leave for now and come back to later, but since we're touching these methods anyway I think we could make the names of some of them slightly better. 

I was initially confused that this returns `None`, because from the signature along I would expect `get_collection` to return the collection. I'm not sure what would be better, but perhaps something along the lines of `ensure_collection_exists()`?

Reply:
Good suggestion, I've made this change now and also changed it to inspect the client error for 404.